modules/Core/queries/admin_users.php

```diff
@@ -6,27 +6,28 @@
     die(json_encode('Unauthenticated'));
 }
 
-$sortColumns = ['username' => 'username', 'nickname' => 'nickname', 'joined' => 'joined'];
+$sortColumns = ['username' => 'username', 'nickname' => 'nickname', 'joined' => 'joined', 'quick_action' => 'quick_action'];
```

Comment:
There is no column named `quick_action` in the `nl2_users` table. What does this do?